README.md

```diff
@@ -1,30 +1,38 @@
 # K10rProject
 
-K10rProject is a helper plugin for Shopware to run migrations and update text snippets. It is highly recommended to use this plugin in combination with our [K10rDeployment](https://github.com/kellerkinderDE/K10rDeployment) plugin in a continuous delivery or automated deployment setup.
-
+`K10rProject` is a helper plugin for Shopware to run migrations and update text snippets. It is highly recommended to use this plugin in combination with our [K10rDeployment](https://github.com/kellerkinderDE/K10rDeployment) plugin in a continuous delivery or automated deployment setup.
 
 ## Usage
 ### Installation
 __This is a project specific plugin, so make sure every project has it's own copy of this base plugin.__
-* Download the ZIP or clone this repository into your `engine/Shopware/Plugins/Local/Core/` folder.
+* Download the ZIP or clone this repository into your `custom/plugins/` folder.
 
 ### Create a Migration
-* Activate the plugin in your local development instance
-* Use the shopware command line to create a new migration: `php bin/console k10r:migration:create`
+* Install and activate `NetcomMigrations` in your local development instance
```

Comment:
Can you add an information on where to get the plugin?

Reply:
Added links in the README :-)